core/src/main/scala-2.13+/cats/data/ZipLazyList.scala

```diff
@@ -29,7 +29,7 @@ object ZipLazyList {
   def apply[A](value: LazyList[A]): ZipLazyList[A] = new ZipLazyList(value)
 
   implicit val catsDataAlternativeForZipLazyList: Alternative[ZipLazyList] & CommutativeApplicative[ZipLazyList] =
-    new Alternative[ZipLazyList] with CommutativeApplicative[ZipLazyList] {
+    new Apply.AbstractApply[ZipLazyList] with Alternative[ZipLazyList] with CommutativeApplicative[ZipLazyList] {
```

Comment:
Sorry for my ignorance, but could you elaborate on the this a little – how adding this `new Apply.AbstractApply[ZipLazyList]` to the instance helps with reducing the the resulting jar size?

I would really appreciate if I could learn some details on this, because this matter is not transparent to me for now.

Thank you!